page-homepage.php

```diff
@@ -221,32 +221,35 @@ function() {
 		</div>
 		<div class="row">
 			<div class="col-sm-6 col-xs-12 install-buttons">
-				<p>Free, Open Source, Community Edition</p>
+				<p>ownCloud Server <?php echo $VERSIONS_SERVER_MAJOR_STABLE; ?>.0</p>
 				<div class="btn-group">
 					<a class="btn btn-primary btn-lg installbtn" href="/install"><i class="icon-cloud-download" ></i> Host ownCloud</br><small>For server owners</small></a>
 					<a class="btn btn-default btn-lg hostingbtn" href="/providers/"><i class="icon-search" ></i> Find hosting</br><small>with a third party</small></a>
 				</div>
-				<a class="btn btn-default btn-lg client-install" href="/install/#desktop"><i class="icon-mobile" ></i> Install Clients</br><small>for desktop and mobile</small></a>
+				<br /><br /><a class="btn btn-default btn-lg client-install" href="/install/#desktop"><i class="icon-mobile" ></i> Install Clients</br><small>for desktop and mobile</small></a>
```

Comment:
I think it looks better... Just that. But maybe it looks different in different browsers...
![eight1](https://cloud.githubusercontent.com/assets/551757/6561650/35935f4c-c693-11e4-9302-756dcbd92937.png)
is what it looks like here. I can either have the three buttons against each other (which I find ugly) or have this (indeed rather big) empty space. least bad, imho.